styles.css

```diff
@@ -0,0 +1,96 @@
+
+body {
+  background: #fff;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.header-image {
+  height: 74px;
+  width: 111px;
+  left: 0;
+  top: 0;
+  position: absolute;
+}
+
+.navbar-item {
+  flex: 1;
+}
+
+.icon {
+  color: #1a2236;
+  position: absolute;
+  height: 14px;
+  width: 20px;
+  right: 16px;
+  top: 19px;
+}
+
+.navbar ul {
+  display: none;
+}
+
+.name {
+  height: 100px;
+  width: 220px;
+  left: 16px;
+  top: 104px;
+  position: absolute;
+  color: #1a2236;
+  font-family: 'Playfair Display', serif;
+  font-size: 50px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 100%;
+  letter-spacing: 0;
+  text-align: left;
+  margin: 0%;
+}
+
+.name h1 {
+  margin: 0;
+}
+
+.intro {
+  position: absolute;
+  height: 126px;
+  width: 288px;
+  left: 16px;
+  top: 224px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  text-align: left;
+}
+
+.cta img {
+  height: 50px;
+  position: absolute;
+  width: 288px;
+  left: 16px;
+  top: 390px;
+  color: #1a2236;
+}
+
+.cta p {
+  height: 26.4705886px;
+  width: 203.6792297px;
+  left: 58px;
+  top: 402px;
+  position: absolute;
```

Comment:
- [ ] Please follow the best practice in CSS, you have repeated rule of position absolute, please fix it.

Reply:
**Changes Required**
- [ ] The css selector property `position` with value of `absolute` is repeated several time. This defies the `Don't Repeat Yourself (DRY)` rule best practice. Please let us create a css class for instance `.position-absolute` to group the repeated patterns. This will make our codebase adhere to the DRY principle.